src/templates/finops-hub/modules/adxTableSchema.kql

```diff
@@ -0,0 +1,3 @@
+.create-merge table Focus (Timestamp:datetime, AvailabilityZone:string, BillingAccountId:string, BillingAccountName:string, BillingCurrency:string, BillingPeriodEnd:datetime, BillingPeriodStart:datetime, ChargePeriodEnd:datetime, ChargePeriodStart:datetime, ChargeType:string, InvoiceIssuerName:string, ProviderName:string, PublisherName:string, Region:string, ResourceId:string, ResourceName:string, ServiceCategory:string, ServiceName:string, SubAccountId:string, SubAccountName:string, AmortizedCost:decimal, BilledCost:decimal)
```

Comment:
Can this be broken out into separate lines for each column? That will make it easier to review when we change things. I'm also wondering if we should sort these in any particular order 🤔 FOCUS decided to sort alphabetically, which is okay, but I tend to think hierarchically. Thoughts?

Reply:
It really doesn't matter, as we use KQL to query, order of columns is irrelevant. Hierarchical sorting is subjective and subject to change, where having it alphabetically sorted makes it easy to find columns (change, add, etc). 
I test if you can have this on separate lines.